models/DataObject/ClassDefinition/Data/StructuredTable.php

```diff
@@ -66,6 +66,20 @@ class StructuredTable extends Data implements ResourcePersistenceAwareInterface,
      */
     public array $rows = [];
 
+    /**
+     * @internal
+     *
+     * @var string|array|null
+     */
+    protected $columnType = null;
```

Comment:
Just quickly checked, these additions (eg. `columnType = null`) are going to be osbolete and removed once this is merged
https://github.com/pimcore/pimcore/pull/13888/files/bb5fed108c73628521e9c8267fdcb33483b66bac?w=1#diff-88c43c7b1020461fcf7abcd0a6e7f4ecd4f0d71466b0944f71ce762877647f33

The rest looks good, not sure if this PR could be merged first and delegate the full clean up in the other PR?